docs/upgrade.md

```diff
@@ -6,164 +6,174 @@ sidebar_label: Upgrade
 
 ------
 
-OpenEBS supports the following three main upgrade paths. Each upgrade has its own significant changes to support and ease use of OpenEBS volume in your k8s cluster.
+OpenEBS supports the upgrade to 0.6 version only from 0.5.3 and 0.5.4.
+For older verions please upgrade to either 0.5.3 or 0.5.4 before upgrading to 0.6. For steps tp upgrade 0.5.3 or 0.5.4 please [cLick](https://v05-docs.openebs.io/) here.
```

Comment:
versions, upgrade to...
For steps tp = To upgrade to 0.5.3 or 0.5.4, [click]...